cypress/integration/advanced-functionalities/admin/system/auth-clients/Create.spec.js

```diff
@@ -11,14 +11,26 @@ describe('Test for creating auth client', () => {
   })
 
   context('Test for creating auth client', () => {
-    it('should be able to create an auth client ', () => {
+    it('should check if some labels and fields are hidden when in create mode', () => {
       cy.visit(adminURL + '/')
       // We wait 3s in order the page to be fully loaded
       cy.wait(3000)
       cy.get('.nav-sidebar').contains('Auth Clients').click()
       // We wait 1s in order the page to be fully loaded
       cy.wait(1000)
       cy.get('[data-test-id="button-new-auth-client"]').click()
+      cy.get('[data-test-id="input-client-secret"]').should('not.exist')
+      cy.get('[data-test-id="input-uri"]').should('not.exist')
+      cy.get('[data-test-id="select-user"]').should('not.exist')
+      cy.get('[data-test-id="button-permissions"]').should('not.exist')
+      cy.get('[data-test-id="button-delete"]').should('not.exist')
```

Comment:
not all labels are checked here - updated and created are missing. Also, check for undelete btn